library/std/convert.qs

```diff
@@ -97,6 +97,36 @@ namespace Microsoft.Quantum.Convert {
         result
     }
 
+    /// # Summary
+    /// Produces a binary representation of a non-negative BigInt, using the
+    /// little-endian representation for the returned array.
+    ///
+    /// # Input
+    /// ## number
+    /// A non-negative BigInt to be converted to an array of boolean values.
```

Comment:
Just noticing an inconsistency of boolean vs. Boolean, but I see both versions throughout the docs strings in the main branch.

Reply:
Updated to use capital letter - Boolean.